src/xlogging.c

```diff
@@ -36,6 +36,18 @@ LOGGER_LOG_GETLASTERROR xlogging_get_log_function_GetLastError(void)
 {
     return global_log_function_GetLastError;
 }
+#else
+LOGGER_LOG_GETERRORNO global_log_function_GetErrorNo = consolelogger_log_with_GetErrorNo;
```

Comment:
>Get [](http://example.com/codeflow?start=42&length=3)

Since the API is named LogErrorNo I think we should also name this global_log_function_ErrorNo (without the Get). To be consistent

Reply:
Of course wherever applicable (xlogging_set_log_function_GetErrorNo also).